src/index.html

```diff
@@ -1,13 +1,281 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap" rel="stylesheet">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__nav">
+            <a href="/">
+              <img
+                class="company-logo"
+                src="./images/MyBiKE.png"
+                alt="company-logo"
+              >
+            </a>
+            <div class="header__icon">
+              <a
+                href="tel:+1 234 5555-55-55"
+                class="header__call"
+              >
+              </a>
+              <a class="header__menu-opener" href="#menu"></a>
+            </div>
+          </div>
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="#">
+              <img
+                class="company-logo"
+                src="./images/MyBiKE-black.svg"
+                alt="company-logo"
+              >
+            </a>
+            <div class="menu__icon">
+              <a
+                href="tel:+1 234 5555-55-55"
+                class="menu__call"
+              >
+              </a>
+              <a class="menu__close" href="#"></a>
+            </div>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a
+                href="#" class="menu__link"
+              >
+                home
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a
+                href="#about-us" class="menu__link"
```

Comment:
make sure all hrefs are functional - i.e. user gets navigated to respective sections
you know, add `id` attribute for sections.